ci/azure-pipelines.yml

```diff
@@ -4,6 +4,20 @@
 # 
 # fabric-chaincode-java azure pipeline configuration.
 # 
+name: RELEASE-$(Date:yyyyMMdd)$(Rev:.rrr)
+
+# Daily build for final quality
+# cf https://crontab.guru/#0_23_*_*_*
+schedules:
+  - cron: "0 23 * * *"
+    displayName: 'Chaincode Java Nightly Driver'
+    branches:
+      include:
+        - master
+        - release-1.4
+    always: true
```

Comment:
Should release-1.4 be in here?  If we have a pipeline for each release branch can't we just schedule the current branch?

Reply:
There's currently only one pipeline setup for each repo - that has to does both.
Separate ones would be an alternative.